docs/developer/index.rst

```diff
@@ -82,6 +80,39 @@ Before you submit a pull request to Bodhi, please ensure that it meets these cri
   `github.com/fedora-infra/bodhi <https://github.com/fedora-infra/bodhi/>`_, or you may e-mail a
   patch to the
   `mailing list <https://lists.fedoraproject.org/archives/list/bodhi@lists.fedorahosted.org/>`_.
+* If your changes contain database migrations, you must add a file named ``summary.migration``
+  in the ``news`` directory, where ``summary`` is a short textual description of the change
+  (or the issue number you're fixing).
+* If it is not present already, add a file in the ``news`` directory named ``username.author``
```

Comment:
Yeah, that was more in case the get-author script makes mistakes, which I've seen it do because some authors use weird or inconsistent email addresses in their git config (like `someone+github@example.com`), so I wanted to offer a way out.

Also, I think it's more and information that's part of the patch than part of the release process commit. I see the get-authors script as a safety net to avoid forgetting someone, not as the plan A for doing things.